airflow/api_fastapi/core_api/datamodels/dag_run.py

```diff
@@ -41,6 +41,12 @@ class DAGRunPatchBody(BaseModel):
     note: str | None = Field(None, max_length=1000)
 
 
+class DAGRunClearBody(BaseModel):
+    """DAG Run serializer for clear endpoint body."""
+
+    dry_run: bool | None = True
```

Comment:
Can never be `None` I believe. Also in the legacy the default value is `False` -> `dry_run = post_body.get("dry_run", False)` I'm not sure we want to change that.


```suggestion
    dry_run: bool = True
```